src/Components/CreateImageWizard/CreateImageWizard.tsx

```diff
@@ -233,8 +234,8 @@ const CreateImageWizard = ({ isEdit }: CreateImageWizardProps) => {
   return (
     <>
       <ImageBuilderHeader inWizard />
-      <section className="pf-l-page__main-section pf-c-page__main-section">
-        <Wizard
+      <PageSection>
+        <Wizard height={550}
```

Comment:
Let's remove this height, it hard codes the wizard height to 550 but what we really want is for the wizard to be able to shrink/grow so the footer is always visible.